todo-with-kuzzle-and-vueJS/src/views/Home.vue

```diff
@@ -0,0 +1,309 @@
+<template>
+  <div id="Home">
+    <NavBar></NavBar>
+    <v-container fluid grid-list-xl text-xs-center>
+      <ManageList
+        :lists="lists"
+        :currentList="currentList"
+        @setCurrentList="setCurrentList"
+        @createList="createList"
+      />
+      <Add class="row col s8" @addTask="addTask"/>
+      <MenuCollection
+        class="row"
+        @deleteSelectedTasks="deleteSelectedTasks"
+        @setSelectedTasksComplete="setSelectedTasksComplete"
+        :completeAllTasks="this.completeAllTasks"
+        :taskLength="this.tasks.length"
+        @setSeeActiveTasks="setSeeActiveTasks"
+        @setSeeCompletedTasks="setSeeCompletedTasks"
+      />
+      <v-list one-line>
+        <template v-for="task in this.tasks">
+          <Task
+            :key="task.index"
+            v-if="task.displayed === true"
+            :complete="task.complete"
+            :index="task.index"
+            :message="task.message"
+            @deleteTask="deleteTask"
+            @setTaskComplete="setTaskComplete"
+          />
+        </template>
+      </v-list>
+    </v-container>
+  </div>
+</template>
+
+<script>
+
+import kuzzle from '../service/Kuzzle';
+
+import MenuCollection from '../components/Menu';
+import Add from '../components/Add';
+import Task from '../components/Task';
+import ManageList from '../components/ManageList';
+import NavBar from '../components/NavBar';
+
+export default {
+  name: 'Home',
+  components: {
+    ManageList,
+    Add,
+    MenuCollection,
+    Task,
+    NavBar
+  },
+  data() {
+    return {
+      lists: [{ text: 'NameOfTheList', value: 'NameOfTheList' }],
+      tasks: [
+        {
+          message: 'messageOfTheTask',
+          index: 0,
+          complete: false,
+          displayed: true
+        }
+      ],
+      completeAllTasks: false,
+      seeActiveTasks: true,
+      seeCompletedTasks: true,
+      currentList: { text: 'build', value: 'build' },
+      indexName: localStorage.getItem('indexName'),
+      success: {
+        position: 'bottomRight'
+      },
+      info: {
+        position: 'bottomLeft'
+      },
+      error: {
+        position: 'topRight'
+      },
+    };
+  },
+  methods: {
+    toasted(type, message) {
+      if (localStorage.getItem('toastsEnabled') === 'false') {
+        return;
+      }
+      switch (type) {
+        case 'info':
+          this.$toast.info(message, 'INFO', this.info);
+          break;
+        case 'error':
+          this.$toast.error(message, 'ERROR', this.error);
+          break;
+        case 'success':
+          this.$toast.success(message, 'SUCCESS', this.success);
+          break;
+      }
+    },
+
+    updateDisplay() {
+      this.tasks.forEach(elem => {
+        elem.displayed = (elem.complete && this.seeCompletedTasks) || (!elem.complete && this.seeActiveTasks);
```

Comment:
This is OK, but I would have probably put this condition in the `v-if` or, better, `v-show`, since this is strictly some display logic. I use to centralize it in the presentation layer. But it works like this as well. It's just that you basically "give up" the reactivity that Vue gives you for free, sticking to the imperative paradigm. This paradigm introduces complexity in many other methods of the component.